src/map/entities/baseentity.cpp

```diff
@@ -202,6 +202,11 @@ uint32 CBaseEntity::GetLocalVar(std::string var)
     return m_localVars[var];
 }
 
+std::map<std::string, uint32> CBaseEntity::GetAllLocalVars()
+{
+    return m_localVars;
```

Comment:
This is returning a full copy of `m_localVars`, no reason you can't return a `&` to it

Reply:
The reason I recommend passing back a reference here is because you have to make copies when you create the eventual Lua table. If you don't do this then you'll create 2x essentially full copies on the way over - just to chuck them away

Reply:
> This is returning a full copy of `m_localVars`, no reason you can't return a `&` to it

Thanks, I updated the PR, hopefully this is ok now.